martin/src/srv/tiles.rs

```diff
@@ -184,9 +185,30 @@ impl<'a> DynTileSource<'a> {
                 }
             }
             if tile.info.encoding == Encoding::Uncompressed {
+                let ordered_encodings = if let Some(prefered) = self.prefered_encodings {
+                    match prefered {
+                        PreferredEncoding::Brotli => [
```

Comment:
I am not too certain if the compiler can optimize these cases on the fly - maybe replace these with 3 statics, like we used to have with `SUPPORTED_ENCODINGS`:  

```rust
static PREFER_BROTLI_ENC: &[HeaderEnc] = &[ ... ];
static PREFER_GZIP_ENC: &[HeaderEnc] = &[ ... ];
```

and in this match statement just do this:

```rust
let ordered_encodings = match self.prefered_encodings {
  Some(PreferredEncoding::Gzip) => PREFER_GZIP_ENC,
  Some(PreferredEncoding::Brotli) | None => PREFER_BROTLI_ENC
};
```

Reply:
Thx!  Much better and efficiency.